### PR TITLE
Feature/add habits list

### PR DESCRIPTION
**Pull request by jvargas98**
## Explain why this change is being made.
As a registered user, i would like to see a list of my current habits so i can track them

## Explain how this is accomplished.
the registered user with saved habits, enters the page and can see the list of habits he has saved.

## How do you manually test this?
- As a registered user with habits saved.
- Visit the page /habits.
- The page should show a list of your habits.

## Screenshots
Screenshot from 2021-04-06 16-17-19
Screenshot from 2021-04-06 16-17-39

### CODE REVIEW

**Comment by ese-varo** [comment]
@jvargas98 I think this pr should point to develop since it is comparing your changes with main and it's taking all the commits already in develop branch

**Review by LoginALee — APPROVED** [review]
Great job Jorge ! :rocket:

**Review by GregVaz — APPROVED** [review]
Great job! :100:

File: app/views/habits/_empty_list.html.haml [hunk 1]
```diff
@@ -0,0 +1 @@
+%h5="You don't have #{word} habits yet."
```
Comment (OmauryMarmolejo) [inline app/views/habits/_empty_list.html.haml]: What is `word`? Maybe something more descriptive?

**Review by OmauryMarmolejo — COMMENTED** [review]
Great job, Jorge! I left a comment.

File: app/views/habits/_empty_list.html.haml (hunk 1 above)
Comment (jvargas98) [reply app/views/habits/_empty_list.html.haml]: Thanks, already changed

**Review by ese-varo — APPROVED** [review]
Well done Vargas! :rocket:

**Review by GregVaz — APPROVED** [review]
Well done!

**Review by OmauryMarmolejo — APPROVED** [review]

**Pull request merged**